l10n_account_translate/README.rst

```diff
@@ -0,0 +1,7 @@
+Description
+===========
+
+This module sets translate=True on the account.account,name field
+and is intended for use by accounting localisation modules.
```

Comment:
L10n_multilang changes a whole set fields into multilingual fields whereas the l10n_account_translate only changes the account.account,name field. 
We are now implementing accounting on OpenERP for 5 years and for most of the customers we turn off the account.account,name translate=True since it's annoying when adding new accounts via 'duplicate'.